action.yaml

```diff
@@ -0,0 +1,29 @@
+name: 'Iter8 Install'
+description: 'Install Iter8 CLI'
+inputs:
+  version:
+    description: "Version of iter8 to install"
+    required: false
+runs:
+  using: 'composite'
+  steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17
+    - name: Install latest version of Iter8 CLI
+      run: | 
+        OS=$(echo ${{ runner.os }} | tr '[:upper:]' '[:lower:]')
+        ARCH=$(echo ${{ runner.arch }} | sed 's/X86/386/' | sed 's/X64/amd64/')
```

Comment:
What happens if `runner.os` or `runner.arch` values happen to be  values for  which we do not produce binary assets?
***
I confess I don't know bash well enough to understand the `OS` and `ARCH` definitions.

Reply:
This is better documented now. The action does better error checking on download to provide a better explanation of any failure. But no assumption is made of what are valid values so that no changes should be necessary if new artifacts are added in the future.